Header header

# The symbolic spatial information observed
string ssi

# The ID of the tag which was observed
int32 tag_id

# The location the symbolic spatial information was observed
geometry_msgs/Pose location
